src/Compiler.php

```diff
@@ -5054,11 +5067,12 @@ public function addParsedFile($path)
      *
      * @api
      *
+     * @deprecated
```

Comment:
we should add the `@trigger_error('...', E_USER_DEPRECATED);` to report the deprecation.

Reply:
fixed